Sources/Server/Router.swift

```diff
@@ -5,30 +5,43 @@ import Routes
 
```

Comment:
Feels like this new format would make it _a lot_ easier to add `OPTIONS` for every route 😜